Src/PerformanceCollector/Filtering/CollectionConfiguration.cs

```diff
@@ -0,0 +1,327 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse.Helpers;
+
+    internal class CollectionConfiguration
+    {
+        private readonly CollectionConfigurationInfo info;
+
+        private readonly List<OperationalizedMetric<RequestTelemetry>> requestTelemetryMetrics = new List<OperationalizedMetric<RequestTelemetry>>();
+
+        private readonly List<OperationalizedMetric<DependencyTelemetry>> dependencyTelemetryMetrics =
```

Comment:
why do you keep a list per type? Would it be better to make configuration more generic?